scipy/stats/_stats_py.py

```diff
@@ -10261,7 +10479,6 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     """
     return _cdf_distance(1, u_values, v_values, u_weights, v_weights)
 
-
```

Comment:
PEP8 recommends two blank lines between function definitions, so please undo this change.

Reply:
Done!